session_test.go

```diff
@@ -1870,7 +1870,6 @@ func (s *testSessionSuite) TestCastTimeToDate(c *C) {
 }
 
 func (s *testSessionSuite) TestSetGlobalTZ(c *C) {
-	defer testleak.AfterTest(c)()
```

Comment:
Why remove this?

Reply:
This line will cause an error in leak test, all the cases should be checked in Setup and Teardown in TestSuite.